SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -11,9 +11,70 @@
 #include "EngineClasses/SpatialPackageMapClient.h"
 #include "SpatialGDKSettings.h"
 #include "SpatialConstants.h"
+#include "UObject/UObjectIterator.h"
 
 DEFINE_LOG_CATEGORY(LogInterestFactory);
 
+namespace
+{
+static TMap<UClass*, float> ClientInterestDistancesSquared;
```

Comment:
If we're running in the editor, this won't get reset between runs.

Reply:
https://improbableio.atlassian.net/browse/UNR-1621